app/assets/javascripts/dashboard.js

```diff
@@ -156,10 +168,35 @@ MinionPoller = {
     masterHtml = '<input name="roles[master][]" id="roles_master_' + minion.id +
       '" value="' + minion.id + '" type="radio" disabled="" ' + checked + '>';
 
+    statusText = ''
+
+    switch(minion.update_status) {
+      case 1:
+        switch (minion.highstate) {
+          case "applied":
+            statusHtml = '<i class="fa fa-arrow-circle-up text-info fa-2x" aria-hidden="true"></i>';
+            statusText = 'Update Available'  
+            break;
+          case "failed":
+            statusHtml = '<i class="fa fa-arrow-circle-up text-warning fa-2x" aria-hidden="true"></i>';
+            statusText = 'Update Failed - Retryable'
+            break;
+          case "pending":
+            statusText = 'Update in progress'
+            break;
+        }
+        break;
+      case 2:
```

Comment:
No - it means the `transactional-update` has failed - which needs manual intervention